test/cpp/jit/test_gpu.cpp

```diff
@@ -2206,28 +2207,86 @@ TEST(NVFuserTest, FusionBCastConcretizeRfactor_CUDA) {
 
 namespace {
 
-void checkIdProvedEquivalent(
+void checkIdMapped(
+    ComputeAtRootDomainMap& root_map,
+    TensorView* v0,
+    IterDomain* id0,
+    TensorView* v1,
+    IterDomain* id1,
+    bool should_map) {
+  if (should_map) {
+    TORCH_CHECK(root_map.canMap(v0->domain(), id0, v1->domain(), id1));
+  } else {
+    TORCH_CHECK(!root_map.canMap(v0->domain(), id0, v1->domain(), id1));
+  }
+}
+
+void checkIdMapped(
+    ComputeAtRootDomainMap& root_map,
     TensorView* v0,
     int a0,
     TensorView* v1,
     int a1,
-    bool should_prove) {
-  if (should_prove) {
-    TORCH_CHECK(IterDomain::proveEquivalent(v0->axis(a0), v1->axis(a1)));
-  } else {
-    TORCH_CHECK(!IterDomain::proveEquivalent(v0->axis(a0), v1->axis(a1)));
+    bool should_map) {
+  return checkIdMapped(
+      root_map,
+      v0,
+      v0->getRootDomain()[a0],
+      v1,
+      v1->getRootDomain()[a1],
+      should_map);
+}
+
+void checkIdMapped(
```

Comment:
Looks like this variant is always called with
```.cpp
(v0, v0->getRootDomain(), v1, v1->getRootdomain() ...
```

Should we just remove the root options on this function?

Reply:
I believe I've removed the one that was not used.